Modules/Bridge/NumPy/wrapping/itkPyVectorContainer.wrap

```diff
@@ -79,6 +82,7 @@ itk_wrap_class("itk::PyVectorContainer")
   endforeach()
   # used in FastMarchingExtensionImageFilter
   itk_wrap_template("${ITKM_UI}${ITKM_VUC1}"    "${ITKT_UI},${ITKT_VUC1}")
+    set(ElementType UC)
     set(PyVectorContainerTypes ${ITKM_UI}${ITKM_VUC1})
```

Comment:
@thewtex Same question as above -- why is `ElementType` set to `unsigned char` while `PyVectorContainer` is wrapped for elements of type `itk::Vector< unsigned char, 1>`?

Reply:
`unsigned char`, i.e. `UC` is the element type of `itk::Vector< unsigned char, 1 >`.